.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            // Use IntelliSense to find out which attributes exist for C# debugging
+            // Use hover for the description of the existing attributes
+            // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+            "name": ".NET Core Launch (web)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            // If you have changed target frameworks, make sure to update the program path.
+            "program": "${workspaceFolder}/src/Point/bin/Debug/net6.0/Point.dll",
+            "args": ["--local-id=99W1", "--local-rasta-id=97", "--remote-id=INTERLOCKING", "--remote-endpoint=http://localhost:50051"],
```

Comment:
Let's not commit the launch.json and tasks.json files since they can be easily generated using VS Code and tend to include information that are specific to the local setup of individual developers.

Reply:
OK